README.md

```diff
@@ -1,2 +1,8 @@
 # dogs
 Formidable Dogs
+
+## Contributing
+This repository is configured using [semantic-release](https://github.com/semantic-release/semantic-release) which automates the whole package release workflow including: determining the next version number, generating the release notes and publishing the package.
```

Comment:
Can we add a simple example of how "inputs" translate to a version number choice and the resulting published package?